<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageDialog</class>
 <widget class="QDialog" name="PageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>页面设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>当前页面</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="10" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>高</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>页面设置</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>页面尺寸</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>宽</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6" colspan="4">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4" colspan="2">
        <widget class="QDoubleSpinBox" name="CurrentPageSizeWidth"/>
       </item>
       <item row="1" column="12" colspan="2">
        <widget class="QDoubleSpinBox" name="CurrentPageSizeHeight"/>
       </item>
       <item row="1" column="14">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>选择尺寸</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="CurrentSetWorkingAreaChecked">
         <property name="text">
          <string>设置工作区域</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>工作区域尺寸</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="CurrentWorkingAreaWidthText">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>宽</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3" colspan="2">
        <widget class="QDoubleSpinBox" name="CurrentWorkingAreaWidth"/>
       </item>
       <item row="10" column="1" colspan="4">
        <widget class="QCheckBox" name="SpecificPageCheckBox">
         <property name="text">
          <string>特定页数</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">AppliedRange</string>
         </attribute>
        </widget>
       </item>
       <item row="4" column="14">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="15">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="5" colspan="3">
        <widget class="QLabel" name="PageNum">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>页码</string>
         </property>
        </widget>
       </item>
       <item row="10" column="8" colspan="7">
        <widget class="QLineEdit" name="SpecificPage"/>
       </item>
       <item row="5" column="6" colspan="3">
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="4">
        <widget class="QCheckBox" name="CurrentPageCheckBox">
         <property name="text">
          <string>当前页</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">AppliedRange</string>
         </attribute>
        </widget>
       </item>
       <item row="4" column="11" colspan="3">
        <widget class="QDoubleSpinBox" name="CurrentWorkingAreaHeight"/>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>水平</string>
         </property>
        </widget>
       </item>
       <item row="4" column="6" colspan="3">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="5" column="11" colspan="3">
        <widget class="QDoubleSpinBox" name="CurrentWorkingAreaY"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>应用于</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>工作区域位置</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="4">
        <widget class="QCheckBox" name="AllPagesCheckBox">
         <property name="text">
          <string>所有页</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">AppliedRange</string>
         </attribute>
        </widget>
       </item>
       <item row="5" column="14">
        <widget class="QLabel" name="label_17">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="4" column="10">
        <widget class="QLabel" name="CurrentWorkingAreaHeightText">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>高</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3" colspan="2">
        <widget class="QDoubleSpinBox" name="CurrentWorkingAreaX"/>
       </item>
       <item row="9" column="1" colspan="4">
        <widget class="QCheckBox" name="PageRangeCheckBox">
         <property name="text">
          <string>页面范围</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">AppliedRange</string>
         </attribute>
        </widget>
       </item>
       <item row="9" column="11" colspan="2">
        <widget class="QLabel" name="PageRange2">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>到</string>
         </property>
        </widget>
       </item>
       <item row="9" column="6">
        <widget class="QLabel" name="PageRange1">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>从</string>
         </property>
        </widget>
       </item>
       <item row="9" column="7" colspan="4">
        <widget class="QSpinBox" name="PageRangeLowerBound"/>
       </item>
       <item row="9" column="13">
        <widget class="QSpinBox" name="PageRangeUpperBound"/>
       </item>
       <item row="9" column="14">
        <widget class="QLabel" name="PageRange3">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>页</string>
         </property>
        </widget>
       </item>
       <item row="5" column="9" colspan="2">
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>垂直</string>
         </property>
        </widget>
       </item>
       <item row="11" column="4">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2" colspan="4">
        <widget class="QComboBox" name="CurrentSelectPageSize">
         <item>
          <property name="text">
           <string>A4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Letter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Customized</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>默认页面</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="5">
        <widget class="QDoubleSpinBox" name="DefaultPageSizeHeight"/>
       </item>
       <item row="4" column="5">
        <widget class="QDoubleSpinBox" name="DefaultWorkingAreaHeight"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_31">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>工作区域位置</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QComboBox" name="DefaultSelectPageSize">
         <item>
          <property name="text">
           <string>A4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Letter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Customized</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="label_33">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QLabel" name="label_23">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>垂直</string>
         </property>
        </widget>
       </item>
       <item row="5" column="6">
        <widget class="QLabel" name="label_24">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QDoubleSpinBox" name="DefaultWorkingAreaX"/>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_30">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>高</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_26">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_20">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>宽</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_18">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>页面尺寸</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_32">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>水平</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QLabel" name="label_21">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>默认页面设置</string>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="QDoubleSpinBox" name="DefaultWorkingAreaY"/>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="label_27">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="DefaultPageSizeWidth"/>
       </item>
       <item row="4" column="2">
        <widget class="QDoubleSpinBox" name="DefaultWorkingAreaWidth"/>
       </item>
       <item row="4" column="4">
        <widget class="QLabel" name="label_28">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>高</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>选择尺寸</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_22">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>工作区域尺寸</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_29">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>宽</string>
         </property>
        </widget>
       </item>
       <item row="4" column="6">
        <widget class="QLabel" name="label_25">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="DefaultSetWorkingAreaChecked">
         <property name="text">
          <string>设置工作区域</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="OkAndCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="AppliedRange">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
